src/components/camera.js

```diff
@@ -128,8 +153,8 @@ module.exports.Component = registerComponent('camera', {
     var camera = this.camera;
     var system = this.system;
     if (!oldData || oldData.userHeight !== data.userHeight) {
-      this.removeHeightOffset();
-      this.addHeightOffset();
+      this.userHeightOffset = oldData.userHeight || 0;
```

Comment:
when you have scene where you would constantly update `userHeight` while you don't want to be applied when in VR with headset, but I guess when you exit-vr you would like to "restore" userHeight to what it was changed not to when you entered into VR - right?